libmachine/libmachine.go

```diff
@@ -152,3 +152,13 @@ func (api *Client) performCreate(h *host.Host) error {
 	return nil
 
 }
+
+func sendCrashReport(err error, api *Client, host *host.Host) {
+	if host.DriverName == "virtualbox" {
+		// getting BaseDriver.ResolveStorePath here sounds like hell...
+		vboxlogPath := filepath.Join(api.Filestore.Path, "machines", host.Driver.GetMachineName(), host.Driver.GetMachineName(), "Logs", "VBox.log")
+		crashreport.SendWithFile(err, "api.performCreate", host.DriverName, "Create", vboxlogPath)
```

Comment:
Instead of having a separate method `SendWithFile`, why not have a method like `crashreport.AttachVBoxLog` which does that operation on the `metaData`, and call that here, allowing it to always fall through to the `crashreport.Send`?  Would reduce the number of possible code paths.